### DISCUSSION TITLE
Release Note for 1.34.0 is missing from kiali.io

### DISCUSSION THREAD

**Discussion by ethernoy** [kiali/kiali#3992] [General]
Can't find release note for 1.34.0 after docker image is available from quay.io. https://kiali.io/news/release-notes/

**Comment by lucasponce** [comment]
True, it seems there was some issue with the CI process to publish, I'm sure it will be fixed during the day.
Thanks @ethernoy !

**Reply by jshaughn** [reply]
Unfortunately I am the CI process for release notes, will publis ASAP.

**Comment by jshaughn** [comment]
Published: https://kiali.io/news/release-notes/
